odsApi_versioned_docs/version-7.3/how-to-guides/how-to-load-the-ods-with-sample-xml-data-using-bulk-load-client-utility.mdx

```diff
@@ -43,14 +43,14 @@ utility.
 
   ```bash
   mkdir <YourInstallFolder>
-  dotnet tool install EdFi.Suite3.BulkLoadClient.Console --version 7.2.413 --tool-path <YourInstallFolder>
+  dotnet tool install EdFi.Suite3.BulkLoadClient.Console --version 7.3.413 --tool-path <YourInstallFolder>
```

Comment:
This may not need to be changed, but want to verify flags like `--version 7.3.413` are still valid when bumping the documentation from 7.2 to 7.3. 

There are some other instances in the docs where this version tag appears as output. I am highlighting this as a user could "copy paste" this data for use.

Reply:
These versions will be updated after API release and right before documentation release.